pandas/tests/frame/test_convert_to.py

```diff
@@ -179,3 +179,11 @@ def test_to_records_with_unicode_index(self):
             .to_records()
         expected = np.rec.array([('x', 'y')], dtype=[('a', 'O'), ('b', 'O')])
         tm.assert_almost_equal(result, expected)
+
+    def test_to_records_with_unicode_column_names(self):
+        # Issue #11879. to_records used to raise an exception when used
+        # with column names containing non ascii caracters in Python 2
+        try:
```

Comment:
Isn't this being fixed by this PR? If so, remove try-except block and compare the result with the expected result.